docs/symptom-survey/coding.md

```diff
@@ -305,6 +310,14 @@ survey, even if they have already taken it before:
 > have participated before. Completing the survey again will help us understand
 > how the situation is changing.
 
+It also more clearly states that the data may be shared and aggregates publicly
+released:
+
+> The de-identified results of this survey may be used for our future studies or
+> shared with other investigators for their research studies. Results published
```

Comment:
```suggestion
> shared with other investigators for their research. Results published
```
Repeating studies felt redundant.

Reply:
This text is part of the IRB-approved consent text, so while I agree with your suggestions, I don't think I can change it -- this is what's in Wave 5 and already went out